### ISSUE TITLE
Documentation about the BCrypt issue under macOS 10.15

### ISSUE THREAD

**Issue by mcritz** [BinaryBirds/feather#3]
Hi. I’m running from the macOS 11 command line using `swift run` and followed the README.md instructions.

Visiting login and entering user: feather@binarybirds.com with password `FeatherCMS` gets this result:

```
[ INFO ] POST /login/ [request-id: 7454CFA2-BD23-4801-81E0-E95263B6979A]
[1]    43303 segmentation fault  swift run
```

I’ve tried this several times with the same problem. Same results with both SQLite and PostgreSQL. `dropdb` and creating a fresh db has the same result.

**Comment by tib** [comment]
This is a problem caused by BCrypt. The Bcrypt.verify method is crashing on macOS platforms when building from command line. It is a Swift / Vapor related bug, can you try running the app using Xcode 11 or later?

**Comment by mcritz** [comment]
I tried that and ended up with #4

**Comment by tib** [comment]
I'll add some notes to the readme later on.

**Comment by tib** [comment]
This is now fixed in 1.0.0-beta.3.
